docs/deployment_guide/partner_editable/post_deployment.adoc

```diff
@@ -35,6 +35,34 @@ The deployment creates a security group that is used by Firewall Manager only as
 
 Firewall Manager creates a web ACL in each account that's managed by Firewall Manager.
 
+=== AWS Firewall Manager AWS Network Firewall policies
+
+* The Resource type for a Network Firewall policy is always VPC
+
+* Resources allows you to narrow the scope of your policy by specifying resource tags for inclusion. To use tagging, you need to first tag your resources. In Your VPCs, you use Resource tags with **key**: **__sumo-awsqs-nf-policy__** and **value**: **__network-firewall__**. For more information about tagging your resources, see https://docs.aws.amazon.com/awsconsolehelpdocs/latest/gsg/tag-editor.html[Working with Tag Editor]
```

Comment:
[Firewall Manager Network Firewall Policies](https://docs.aws.amazon.com/waf/latest/developerguide/waf-policies.html) allow Firewall Manager to manage AWS Network Firewall firewalls for your Amazon Virtual Private Cloud VPCs across your organization in AWS Organizations. For each account and VPC that's within policy scope, Firewall Manager creates a Network Firewall firewall and deploys firewall endpoints to VPC subnets, to filter network traffic. 
To narrow the scope of your policy, you can tag your VPCs with **key**: **__sumo-awsqs-nf-policy__** and **value**: **__network-firewall__**. For more information about tagging your resources, see https://docs.aws.amazon.com/awsconsolehelpdocs/latest/gsg/tag-editor.html[Working with Tag Editor].